tests/apitest/src/CoreGraphics/CGDisplay.cs

```diff
@@ -36,24 +36,24 @@ public void CGDisplayShouldGetDimensions ()
 			Assert.IsFalse (bounds.IsEmpty);
 
 			var width = CGDisplay.GetWidth (mainDisplayId);
-			Assert.Greater (0, width);
+			Assert.Greater (width, 0);
```

Comment:
same comment as before, this needs a `message` argument so it can be distinguished from other assert when debugging symbols are missing/broken